src/index.ts

```diff
@@ -22,6 +26,19 @@ export { Radio } from './components/forms/Radio/Radio'
 export { Textarea } from './components/forms/Textarea/Textarea'
 export { TextInput } from './components/forms/TextInput/TextInput'
 
+/** Header Components */
+export { ExtendedNav } from './components/header/ExtendedNav/ExtendedNav'
+export { Header } from './components/header/Header/Header'
+export { MegaMenu } from './components/header/MegaMenu/MegaMenu'
+export { Menu } from './components/header/Menu/Menu'
+export { NavCloseButton } from './components/header/NavCloseButton/NavCloseButton'
+export { NavList } from './components/header/NavList/NavList'
+export { NavMenuButton } from './components/header/NavMenuButton/NavMenuButton'
+export { PrimaryNav } from './components/header/PrimaryNav/PrimaryNav'
+export { Title } from './components/header/Title/Title'
+
+export { Search } from './components/Search/Search'
```

Comment:
Was this a hotfix to the release branch only? I only notice because I've been having conflicts in my work on `index.ts` so I noticed this code and commit isn't on `develop`.

Reply:
Also when there are changes only to the release branch that aren't semantic versioning changes -- Let's call it out in the PR.

Reply:
yup it was a hotfix I noticed.. adding to the PR!